### PR TITLE
feat: Compare string arrays

### PR DESCRIPTION
**Pull request by andrewi-wd**
## Changes

Add new isomorphic util to compare two string arrays

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns true if the two arrays are equal, false otherwise.
+ * The arrays are considered equal if they have the same length, same order and the elements at each index are equal.
+ */
+export function areStringArraysEqual(a: string[], b: string[]): boolean {
```
Comment (kibertoad) [inline packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts:5]: can you also add documentation for this method to readme.md?

Comment (andrewi-wd) [reply packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts:5]: Sure! Done in https://github.com/lokalise/shared-ts-libs/pull/280/commits/2698bf73566ae3913fa349bd75c9285c8a55694c

File: packages/app/universal-ts-utils/README.md [hunk 2]
```diff
@@ -4,3 +4,16 @@ Isomorphic general-purpose TS utils
 
 ## Overview
 
+
+### String array equality
+
+This package also provides a function to compare string arrays:
```
Comment (kibertoad) [inline packages/app/universal-ts-utils/README.md:10]: can you clarify the rules for comparison? "same elements in same order, compared by strict equality"

**Review by kibertoad — APPROVED** [review]

**Review by dariacm — APPROVED** [review]

File: packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts (hunk 1 above)
Comment (CarlosGamero) [inline packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts:5]: 🟢 Nit: Wondering if would be useful to extend it to be used with other basic types (eg: numbers)

**Pull request merged**